client/app.js

```diff
@@ -148,6 +148,196 @@ tie.constant('PREREQ_CHECK_TYPE_BAD_IMPORT', 'badImport');
  * @constant
  */
 tie.constant('PREREQ_CHECK_TYPE_GLOBAL_CODE', 'globalCode');
+tie.constant('PREREQ_CHECK_TYPE_WRONG_LANG', 'wrongLang');
+
+/**
+ * Dictionary of wrong language detection errors and their related information
+ *
+ * @type {{}}
+ */
+tie.constant('WRONG_LANGUAGE_ERRORS', {
+  python: [
```

Comment:
Consider compactifying a bit, similar to what we did [here](https://github.com/google/tie/pull/356/files#diff-b2454bae671216e787350ab0d8bc2380R158).